imports/ui/admin/Admin.js

```diff
@@ -21,48 +21,50 @@ Template.Admin.events({
 
     'submit .new-player'(event) {
         event.preventDefault();
+        
+        if (confirm("The user will the above information will be added to the database.")) {
```

Comment:
Typo: will -> with

Also could this be phrased more conventionally?

"Are you sure you wish to add this user?"

Reply:
The options are "Ok" and "Cancel", instead of "Yes" and "No".